modules/cnf-performance-profile-creator-arguments.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+
+[id="performance-profile-creator-arguments_{context}"]
+=  Performance Profile Creator arguments
+
+.Performance Profile Creator arguments
+["cols="30%,5%,65%",options="header"]
+|===
+| Argument | Values | Description
+
+| --mcp-name
+a| worker-cnf (default)
+|MCP name corresponding to the target machines (required)
+
+|--reserved-cpu-count
+a| 0 (default)
+| Number of reserved CPUs (required)
+
+| --rt-kernel
+a| true (default)
+| Enable Real Time Kernel (required)
+
+| --must-gather-dir-path
+a| must-gather (default)
+| Must gather directory path (required)
+
+| --split-reserved-cpus-across-numa
+a| true
+| Split the reserved CPUs across NUMA nodes
+
+|--disable-ht
+a| false
```

Comment:
We should specify the default is indeed false and add a warning to the user: The user is not expected to disable HT in BIOS if this arg is set to true, the HT disabling would be done as a kernel command line argument